This patch is part of a series of patches for speech-tools for
Debian GNU/Linux, set backtrace to static

Index: b/siod/slib.cc
===================================================================
--- a/siod/slib.cc
+++ b/siod/slib.cc
@@ -128,7 +128,7 @@
 long interrupt_differed = 0;
 LISP oblistvar = NIL;
 LISP current_env = NIL;
-LISP backtrace = NIL;
+static LISP backtrace = NIL;
 LISP restricted = NIL;
 LISP truth = NIL;
 LISP eof_val = NIL;
